examples/chatgpt/gpt_actions_library/gpt_action_mongodb_vector.md

```diff
@@ -0,0 +1,356 @@
+# GPT Action Library: MongoDB Atlas Vector Search
+
+## Introduction
+
+
+This page provides an instruction & guide for developers building a GPT Action for a specific application. Before you proceed, make sure to first familiarize yourself with the following information:
+
+- [Introduction to GPT Actions](https://platform.openai.com/docs/actions)
+- [Introduction to GPT Actions Library](https://platform.openai.com/docs/actions/actions-library)
+- [Example of Building a GPT Action from Scratch](https://platform.openai.com/docs/actions/getting-started)
+
+This particular GPT Action provides an overview of how to connect to MongoDB Atlas. This Action takes a user’s question and generates MQL to perform a semantic search on a particlar collection to answer the user’s question. It uses Google Cloud Run Functions and OpenAI's Embeddings API to transform the user query to an embedding that will be used as a query parameter for the semantic search.
+
+### Value + Example Business Use Cases
+
+**Value**: Users can now leverage ChatGPT's natural language capability to connect directly to MongoDB Atlas and perform a semantic search. 
+
+**Example Use Cases**:
+
+- A user needs to perform a vector search in MongoDB, but needs a middleware app between ChatGPT and Google Cloud SQL
+
+- A user has thousands of files and has gone through the process of vectorizing the files and storing them in MongoDB
+
+## Application Information
+
+### Application Key Links
```

Comment:
I guess we're trying to remove those links or have them as hyperlink in the description to avoid verbosity. The Canvas LMS cookbook as been revamped to show this: https://cookbook.openai.com/examples/chatgpt/gpt_actions_library/gpt_action_canvas / https://github.com/openai/openai-cookbook/blob/main/examples/chatgpt/gpt_actions_library/gpt_action_canvas.md